src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf),
+    url(/fonts/Avenir-Heavy.ttf),
+    url(/fonts/Avenir.ttc);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding-left: 8px;
+  padding-top: 20px;
+  padding-right: 8px;
+}
+
+.form__search--big {
+  padding-top: 23px;
+  padding-bottom: 23px;
+}
+
+.form__search--big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0,0,0, 0.25);
+}
+
+.form__input--big::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.form__search--small {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.form__search--small:focus {
+  font-family: "Avenir", sans-serif;
+  font-weight: 900;
+}
+
+.form__search--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61,78,97, 0.2);
+}
+
+.form__input--small::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.form__search {
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61,78,97, 0.1);
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.form__search:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+}
+
+.form__icon--big {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: inline-block;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-right: 17px;
+  flex-shrink: 0;
+}
+
+.form__icon--small {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: inline-block;
+  width: 11px;
+  height: 11px;
+  margin-right: 9px;
+  margin-left: 13px;
+}
```

Comment:
you're duplicating your code here.
use common class to set properties and override for small one those you need.

Reply:
for example:

```css
.form__icon {
// styles for big icon
}

.fom_icon--small {
// styles that are different from bigger one
}
```